### ISSUE TITLE
add pictures for Phaulacridium vittatum

### ISSUE THREAD

**Issue by klausriede** [sfg-taxonpages/orthoptera#22]
https://sfg-taxonpages.github.io/orthoptera/#/otus/930718/overview

due to many synonyms there are pictures of syntypes available. These should be visible in TP

**Comment by typophyllum** [comment]
There are altogehter 17 pictures in the old OSF and the same 17 pictures in the new version. That they are shown in arbitrary sequence is probably the real problem.

**Comment by mjy** [comment]
What questions can you not answer because of the order or look or feel of
something of the page?

M

**Comment by klausriede** [comment]
Actually I do not see any of these pictures here
https://sfg-taxonpages.github.io/orthoptera/#/otus/930718/overview

**Comment by klausriede** [comment]
where is the picture 
![grafik](https://github.com/sfg-taxonpages/orthoptera/assets/12816652/ae585e2d-81f3-4117-939c-3c47300ff928)

**Comment by klausriede** [comment]
better. But where comes the DORSA specimen from? 
![grafik](https://github.com/sfg-taxonpages/orthoptera/assets/12816652/474da6e9-5a8d-47a1-b710-dbbb1ef21179)

This should be a german Institution not appearing in the list. Origin of Photograph unclear

**Comment by typophyllum** [comment]
In large view the source is displayed. As I mentioned elsewhere, we need a separate tab for the specimens and their pictures.

![imagen](https://github.com/sfg-taxonpages/orthoptera/assets/33210201/90781f6a-1721-4070-9af4-0b948dd3847f)

**Comment by klausriede** [comment]
yes, and info in the picture after download. Picture metadata are nonexistent after download, but should be incorporated following a ccby scheme

**Comment by mjy** [comment]
@klausriede can you please link to ccby scheme in question please?  Could you also link to other resources that provide this information when you download an image?  Does this come in a zip file?

**Comment by klausriede** [comment]
I understood that pictures in OSF are cc-by at least those we provided via DORSA. I do not know any resources which provide this information. It is a general failure in managing multimedia data. But as we are migrating it would be nicetohave....

**Comment by mjy** [comment]
But how, technically?!  We are surely not the first to encounter and solve this problem are we? :smile:

**Comment by klausriede** [comment]
@LocoDelAssembly  I think that should be discussed at TDWG and the FAIR Data meetings. In fact as far as I know nobody does it, and I think it is a desideratum. It might be the reason that some contributors prefer to provide links in OSF instead of pictures. Integrating text into the picture might be a workaround, usually there is space enough . Hernan, didnt you send around a picture with a bottomline of text recently?

**Comment by klausriede** [comment]
PS: you would be pioneers in solving this problem at least in the lifesciences domain (there are NFDIs in art...)
